macros/calibration/NEXT100_cal.config.mac

```diff
@@ -0,0 +1,34 @@
+## ----------------------------------------------------------------------------
+## nexus | NEXT100_cal.config.mac
+##
+## Configuration macro to simulate decays of the Th calibration source
+## from one of the vessel ports in the NEXT-100 detector.
+##
+## The NEXT Collaboration
+## ----------------------------------------------------------------------------
+
+##### VERBOSITY #####
+/run/verbose 0
+/event/verbose 0
+/tracking/verbose 0
+
+/process/em/verbose 0
+
+##### GEOMETRY #####
+/Geometry/Next100/elfield false
+/Geometry/Next100/max_step_size 1. mm
+/Geometry/Next100/th_source next100 #next_white
```

Comment:
I understand that this can be confusing, but the `next100`/`next_white` parameter refers to the kind of source to be used. Both of them will be used in NEXT100, and I just call the old one "the NEXT-White source" and the new one "the NEXT100 source". If you have any suggestions to make this clearer, please, tell me.

Reply:
Ok, gotcha. It's clear once you know... but I agree it's prompt confusion at first glance. Not having any brilliant advice myself, I think we should avoid the NEW/N100 naming. Maybe by the year or something of that sort?

Reply:
I tried with old/new, is that better?

Reply:
I believe so. It can go like that for now. My only concern is that it is a temporal patch since we may acquire more sources in the future, and this convention runs short. But, again, it's fine with me for the scope of this PR.